<?xml version="1.0"?>
<launch>
  <arg name="aruco_dictionary" default="DICT_4X4_100" doc="Name of aruco dictionary"/>
  <arg name="aruco_id" default="0" doc="id of the aruco code"/>
  <arg name="aruco_description_name" default="$(arg aruco_dictionary)_id$(arg aruco_id)_description" doc="Name of the ros parameter storing the aruco description"/>
  <arg name="aruco_name" default="$(arg aruco_dictionary)_id$(arg aruco_id)"/>

  <!--Environment description to param server-->
  <param name="$(arg aruco_description_name)" command="$(find xacro)/xacro '$(find aruco_description)/urdf/general_aruco.urdf.xacro' aruco_dictionary:=$(arg aruco_dictionary) aruco_id:=$(arg aruco_id) aruco_name:=$(arg aruco_name)"/>

</launch>
